Tasks/VsTest/vstest.ts

```diff
@@ -49,6 +49,7 @@ try {
     var useNewCollectorFlag = tl.getVariable('tia.useNewCollector');
     var isPrFlow = tl.getVariable('tia.isPrFlow');
     var vsTestVersionForTIA: number[] = null;
+	var ignoreVstestFailure: string = tl.getVariable("vstest.ignoretestfailures");
```

Comment:
Agreed. However, we are going ahead with a variable because we don't want this to be visible to all consumers. We want to pilot this in MSEng before going ahead and exposing this as a user provided input. In Phase 2 of reliability efforts (2017 early), this task will be updated to rerun failed tests at which time we will make the decision on whether this is useful for 3rd party and would be exposed as a user input or hidden (consumable just for mseng).